api/datasets.go

```diff
@@ -563,32 +563,15 @@ func loadFileIfPath(path string) (file *os.File, err error) {
 	return os.Open(path)
 }
 
-// default number of entries to limit to when reading
-// TODO - should move this into lib
-const defaultDataLimit = 100
-
 // DataResponse is the struct used to respond to api requests made to the /data endpoint
 // It is necessary because we need to include the 'path' field in the response
 type DataResponse struct {
 	Path string          `json:"path"`
 	Data json.RawMessage `json:"data"`
 }
 
-func (h DatasetHandlers) bodyHandler(w http.ResponseWriter, r *http.Request) {
-	d, err := DatasetRefFromPath(r.URL.Path[len("/body"):])
-	if err != nil {
-		util.WriteErrResponse(w, http.StatusBadRequest, err)
-		return
-	}
-
+func makeGetBodyParams(r *http.Request, readOnly bool, path string) (*lib.GetParams, error) {
```

Comment:
`getParamsFromRequest`? I think it's ok to define the function in generic terms, and add a comment that says something like:
```go
// getParamsFromRequest creates getParams from a request. It's currently only used for paginating dataset bodies
```